packages/net/http/_client_connection.pony

```diff
@@ -98,7 +99,7 @@ actor _ClientConnection is HTTPSession
         if node()? is request then
           try (_conn as TCPConnection).dispose() end
           _conn = None
-          node.pop()?
+          node.>remove().pop()?
```

Comment:
@peterzandbergen would you mind putting spaces around the `.>` operator to match the style guide?

Reply:
will fix it